### PR TITLE
[SMALLFIX]correct the typo

### PR DESCRIPTION
**Pull request by entslscheia**
https://alluxio.atlassian.net/browse/ALLUXIO-2300

MG1633025 谷雨

### CODE REVIEW

**Comment by aaudiber** [comment]
ok to test

**Comment by aaudiber** [comment]
Jenkins, add to whitelist

**Comment by entslscheia** [comment]
alluxio-bot, check this please

**Comment by entslscheia** [comment]
alluxio-bot, check this please

File: core/server/src/main/java/alluxio/master/file/options/SetAttributeOptions.java [hunk 1]
```diff
@@ -251,7 +251,7 @@ public int hashCode() {
   @Override
   public String toString() {
     return Objects.toStringHelper(this)
-        .add("mPinned", mPinned)
+        .add("Pinned", mPinned)
```
Comment (jsimsa) [inline core/server/src/main/java/alluxio/master/file/options/SetAttributeOptions.java]: this should say `pinned`

Comment (apc999) [inline core/server/src/main/java/alluxio/master/file/options/SetAttributeOptions.java]: yes, let's keep the string consistent with other fields

**Comment by entslscheia** [comment]
alluxio-bot, check this please

File: core/server/src/main/java/alluxio/master/file/options/SetAttributeOptions.java (hunk 1 above)
Comment (entslscheia) [inline core/server/src/main/java/alluxio/master/file/options/SetAttributeOptions.java]: I have changed this already. Thanks for reminding me.

**Pull request merged**